src/Administration/Resources/app/administration/src/module/sw-order/component/sw-order-new-customer-modal/index.js

```diff
@@ -86,37 +87,6 @@ export default {
             return this.customer !== null ? this.customer.addresses.get(this.customer.defaultBillingAddressId) : null;
         },
 
-        isSameBilling: {
-            get() {
-                if (this.customer === null) {
-                    return true;
-                }
-
-                return this.customer.defaultBillingAddressId === this.customer.defaultShippingAddressId;
-            },
-
-            set(newValue) {
```

Comment:
@nguyenytran Or we could prevent it from triggering twice by adding a condition at the beginning:

```
set(newValue) {
    if (newValue === this.isSameBilling) {
       return;
    }
    ...
}
```